images/entrypoint.sh

```diff
@@ -380,6 +380,11 @@ generateMultusConf
 
 # ---------------------- end Generate "00-multus.conf".
 
+# remove "00-multus.conf" on close.
+if [ "$MULTUS_CONF_FILE" == "auto" ]; then
+  trap "rm -f $CNI_CONF_DIR/00-multus.conf" EXIT
+fi
+
 # Enter either sleep loop, or watch loop...
 if [ "$MULTUS_CLEANUP_CONFIG_ON_EXIT" == true ]; then
```

Comment:
@dougbtv "--cleanup-config-on-exit" is used to set "MULTUS_CLEANUP_CONFIG_ON_EXIT" which keep tracking that the file exists, which is confusing, do u want to change this param to something else, then use it to clean config file 00-multus.conf